### PR TITLE
fix format by using flake8

### PR DESCRIPTION
**Pull request by hirosassa**
## What is this PR?

This PR fixes formatting by using `flake8` command.

I'll implement CI running on GitHub Actions in next PR to reduce reviewers' work load.

### CODE REVIEW

File: src/cli.py [hunk 1]
```diff
@@ -8,23 +8,18 @@
 sys.path.append(os.path.join(app_home, "lib"))
 sys.path.append(os.path.join(app_home, "conf"))
 
-# load config
-from conf.config import Config
+from conf.config import Config  # noqa
+from lib.controller import Controller  # noqa
+from lib.firestore import Firestore  # noqa
+from lib.slack import Slack  # noqa
```
Comment (hirosassa) [inline src/cli.py:14]: to minimize `diff`, skip flake8 check by using `# noqa`

File: src/cli.py [hunk 2]
```diff
@@ -58,39 +53,39 @@ def snapshot(ctx):
     pass
 
 
-@backup.command(help="Backup specified table and fields description")
+@backup.command(name='table', help="Backup specified table and fields description")
 @click.option('--dataset', '-d', required=True)
 @click.option('--table', '-t', required=True)
-def table(table, dataset):
+def backup_table(table, dataset):
```
Comment (hirosassa) [inline src/cli.py:59]: avoid [E811](https://www.flake8rules.com/rules/F811.html) error

File: src/lib/firestore.py [hunk 3]
```diff
@@ -85,17 +86,16 @@ def _copy_collection(self, src_col, dst_col):
     def _copy_doc(self, src_col, dst_col, document_id):
         src_doc_ref = self.firestore_client.collection(src_col).document(document_id)
         src_doc = src_doc_ref.get()._data
-        if src_doc == None:
+        if src_doc is None:
```
Comment (hirosassa) [inline src/lib/firestore.py:89]: This will cause bug.

File: src/lib/firestore.py [hunk 4]
```diff
@@ -20,9 +20,11 @@ def __init__(self, config, logger):
             os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = config.gcp_key_json
 
         cred = credentials.ApplicationDefault()
-        if not firebase_admin._DEFAULT_APP_NAME in firebase_admin._apps:
+        if firebase_admin._DEFAULT_APP_NAME not in firebase_admin._apps:
```
Comment (hirosassa) [inline src/lib/firestore.py:23]: for readability

File: test/HOW_TO_TEST.md [hunk 5]
```diff
@@ -23,4 +23,10 @@ run test
...
 python -m unittest discover test "test_*.py"
-```
\ No newline at end of file
+```
+
+run linter
+
+```
+tox -e lint
+```
```
Comment (hirosassa) [inline test/HOW_TO_TEST.md:32]: document added

File: tox.ini [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+ignore = H301, H306, E252, W605, E121, E123, E126, E226, E24, E704, W503, W504
+max-line-length = 180
+exclude = resource/*,venv/*
+
+[testenv:lint]
+whitelist_externals =
+    flake8
+skip_install = true
+commands =
+    flake8 --exclude=.venv/* --exclude=.tox/* src test {posargs}
```
Comment (hirosassa) [inline tox.ini:24]: `src` and `test` directories are the target of 'flake8'

**Review by fetaro — APPROVED** [review]
LGTM!

**Review by TetsutaroWatanabe — APPROVED** [review]
LGTM

**Pull request merged**